cmd/syncthing/main.go

```diff
@@ -634,6 +634,10 @@ func syncthingMain() {
 		l.Fatalln("Cannot open database:", err, "- Is another copy of Syncthing already running?")
 	}
 
+	protectedConfigFiles := []string{
```

Comment:
I'd call this (and the corresponding attribute) simply "protectedFiles", as it's not only config.